web/packages/teleterm/src/main.ts

```diff
@@ -39,23 +39,15 @@ if (app.requestSingleInstanceLock()) {
   app.exit(1);
 }
 
-function initializeApp(): void {
+async function initializeApp(): Promise<void> {
```

Comment:
Thanks for the refactor. Did you run into problems with the `writeSync` function? Is it used anywhere else other than when quitting the app?